spaceR/spaceR/ViewModels/RocketViewModel.swift

```diff
@@ -28,11 +28,7 @@ class RocketViewModel {
     }
     
     func getRocketImage(index: Int) -> String {
-        if index == 3 {
-            return allRockets[index].images[1] ?? ""
-    } else {
-            return allRockets[index].images[0] ?? ""
-        }
+        return (allRockets[index].images.randomElement() ?? "") ?? allRockets[index].images[0]!
```

Comment:
Some force unwrapping here 😄

Reply:
Okay, i changed it. Please have a look if it better